### DISCUSSION TITLE
Can I use selenium webdriver in allegro

### DISCUSSION THREAD

**Discussion by waldemarmozola16** [allegro/hermes#1691] [Q&A]
I got the error when I want to add cookie: org.openqa.selenium.InvalidCookieDomainException: invalid cookie domain
My source code: https://gist.github.com/waldemarmozola16/0a8b08ea5331628cfca748cb6484b47b

**Comment by piotrrzysko** [comment]
Hello, could you please clarify how your question is related to this project?

**Comment by piotrrzysko** [comment]
Closing due to a lack of an answer. Feel free to reopen it when you can provide more information.
